src/pages/trader-tools/common/_utility.tsx

```diff
@@ -3,18 +3,18 @@ import validation from './_validation'
 type ErrorsType = {
     errors?: null
     symbol?: string
-    volume?: string
+    volume?: string | JSX.Element
     assetPrice?: string
     leverage?: string
-    pointValue?: string
-    takeProfitAmount?: string
-    stopLossAmount?: string
-    commission?: string
-    stake?: string
-    multiplier?: string
-    takeProfitLevel?: string
-    stopLossLevel?: string
-    swapRate?: string
+    pointValue?: string | JSX.Element
+    takeProfitAmount?: string | JSX.Element
+    stopLossAmount?: string | JSX.Element
+    commission?: string | JSX.Element
+    stake?: string | JSX.Element
+    multiplier?: string | JSX.Element
+    takeProfitLevel?: string | JSX.Element
+    stopLossLevel?: string | JSX.Element
+    swapRate?: string | JSX.Element
```

Comment:
how they can be both string or JSX element?

Reply:
This is because of the **returntype** of `numberValidation` function in the `src/pages/trader-tools/common/_validation.tsx`,
Base on the conditions it returns **string (localize)** or **ReactElement (<Localize translate_text="{{fieldName}} is required" values={{ fieldName }} />)**, so the reason for both of these types is that!